packages/cloud-cognitive/src/components/Datagrid/Datagrid/addons/CustomizeColumns/Actions.js

```diff
@@ -7,24 +7,25 @@
  * restricted by GSA ADP Schedule Contract with IBM Corp.
  */
 
-import { Reset16 } from '@carbon/icons-react';
+// import { Reset16 } from '@carbon/icons-react';
 import PropTypes from 'prop-types';
-import { Button, Search } from 'carbon-components-react';
+// import { Button, Search } from 'carbon-components-react';
+import { Search } from 'carbon-components-react';
 import * as React from 'react';
-import keyBy from 'lodash/keyBy';
+// import keyBy from 'lodash/keyBy';
 import { pkg } from '../../../../../settings';
-import { isColumnVisible } from './common';
+// import { isColumnVisible } from './common';
 
 const blockClass = `${pkg.prefix}--datagrid`;
 
 const Actions = ({
   searchText,
-  columns,
-  originalColumnDefinitions,
-  setColumnsObject,
+  // columns,
+  // originalColumnDefinitions,
+  // setColumnsObject,
```

Comment:
if these arguments are no longer used they should be removed

(the less comments and less commented out code, the easier it is to parse the module)